rpc/src/main/scala/proto/annotations.scala

```diff
@@ -38,6 +38,9 @@ package object protocol {
   class message extends StaticAnnotation
 
   class option(val name: String, val value: String, val quote: Boolean) extends StaticAnnotation
+
+  @message
+  case class Empty()
```

Comment:
can this be an `object Empty` instead?

Reply:
I'm afraid it cannot be. We need a valid type (serializable) that we could use as request/response type.